app/components/icon_list/icon_list.css

```diff
@@ -30,6 +30,7 @@
 
 .IconList--row-3 .IconList-element {
   flex: 1 0 30%;
+  gap: 0 calc(2/3 * var(--spacing-unit));
```

Comment:
Before, the gap between the icon and the text and between multiple elements of the icon list where the same. In some cases, this can make it *slightly* more difficult to visually associate which icons and texts belong to each other. Making the gap between the icon and text of an individual elements slightly smaller than the gap between multiple elements helps.

<img width="599" alt="Screen Shot 2021-06-07 at 11 13 02" src="https://user-images.githubusercontent.com/1512805/120995000-4c52ba80-c785-11eb-9ef1-700555421faf.png">

<img width="611" alt="Screen Shot 2021-06-07 at 11 12 33" src="https://user-images.githubusercontent.com/1512805/120995051-570d4f80-c785-11eb-9639-e6fe0429ecb1.png">

(Of course, there are other factors that impact how we associate elements laid out visually, e.g. text alignment.)